package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "date-fp",
-  "version": "3.0.5",
+  "version": "3.0.6",
```

Comment:
That's a fair point. I don't really have a problem jumping to `4.0`. @cullophid What do you think?